packages/angular/src/ng-package.json

```
@@ -8,6 +8,7 @@
 		"@ai-apps/styles",
 		"flatpickr",
 		"@carbon/icons",
-		"@carbon/telemetry"
+		"@carbon/telemetry",
+    "date-fns"
```

Indentation looks off
```suggestion
		"date-fns"
```